spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
```

Comment:
- [ ] Hey there! 👋 Noticed something in the screenshot below – your post requests tests are failing, and it seems to be because of a mismatch in naming conventions. 🚨 In your controller, it's named post_controller, but in your routes, you're referencing posts. 🔄 Let's sync up the naming convention for consistency – either update the controller to posts_controller or adjust the routes to post. 🛠️ This should resolve the issue and get those tests passing again! Thanks for your attention to detail! 

![Screenshot from 2023-12-05 20-14-37](https://github.com/Chimoneg27/Rails-Blog-App/assets/112261237/1c192e16-8e60-40c8-bac3-f227abe378da)

- [ ] A quick and straightforward solution is to rename post_controller.rb to posts_controller.rb. 🔄 After implementing this change, your test cases should pass without any issues. 🚀 I've attached a screenshot below to illustrate that all tests in post_spec.rb are now successful following this modification. 👇 Thanks for addressing this, and happy testing! 🙌

![Screenshot from 2023-12-05 20-19-38](https://github.com/Chimoneg27/Rails-Blog-App/assets/112261237/f3d554a8-8f63-41be-8ce4-9c0889b76754)